proto/tabletmanagerdata.proto

```diff
@@ -549,7 +548,6 @@ message StopReplicationAndGetStatusResponse {
 
   // Status represents the replication status call right before, and right after telling the replica to stop.
   replicationdata.StopReplicationStatus status = 2;
-  bool backup_running = 3;
```

Comment:
A field for this is already present in replicationdata.Status, no need to add another one here which just led to confusion in the previous PR.